nav2_recoveries/plugins/spin.cpp

```diff
@@ -119,10 +119,15 @@ Status Spin::onCycleUpdate()
   action_server_->publish_feedback(feedback_);
 
   double remaining_yaw = abs(cmd_yaw_) - abs(relative_yaw_);
-  if (remaining_yaw <= 0) {
+  if (remaining_yaw < 1e-6) {
     stopRobot();
     return Status::SUCCEEDED;
   }
+  else{
```

Comment:
I know this is probably useful for debugging, but remove this for the PR since this will spam the console